### PR TITLE
Accessibility: Browsing Menu refactoring

### PR DESCRIPTION
**Pull request by mallexxx**
Task/Issue URL: https://github.com/duckduckgo/iOS/issues/1241
Tech Design URL: 
CC: @bwaresiak @Bunn @miasma13 

**Description**:
Browsing Menu refactored to a Modal View Controller using custom Animator; The refactoring moved to a separate PR as a lot of code has been affected
- When displayed, underlaying elements won‘t be displayed by VoiceOver
- Header Buttons (New, Copy, Share, Print) made Button elements, added accessibility action for these custom buttons
- Accessibility label for the "New" button set to "New Tab"
- Menu items type set to Buttons instead of Labels

**Steps to test this PR**:
1. Validate menu works: animations, placement, device rotation, iPad/iPhone
2. Validate VoiceOver controls works (Xcode tools -> Accessibility Inspector; set Simulator in top-left menu and run through the elements of the menu, validate actions and labels)

**Orientation Testing**:

* [ ] Portrait
* [ ] Landscape

**Device Testing**:

* [ ] iPhone
* [ ] iPad

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**

### CODE REVIEW

**Comment by amddg44** [comment]
@mallexxx This is all looking really good so far except for one issue I've come across - when rotating the device the menu doesn't stay anchored to the menu button

https://user-images.githubusercontent.com/91189922/182626131-a9af17e3-1e54-4f28-bc95-864620e241e0.mov

**Comment by amddg44** [comment]
One more small issue for you - on iPad the positioning of the menu is off in both portrait and landscape mode (although it does correct itself when you rotate the device) 

iPadPortrait
iPadLandscape

**Comment by mallexxx** [comment]
@amddg44, should be fixed now!

File: DuckDuckGo/TabViewControllerBrowsingMenuExtension.swift [hunk 1]
```diff
@@ -22,15 +22,16 @@ import Core
 import BrowserServicesKit
 import simd
 
-// swiftlint:disable file_length
```
Comment (amddg44) [inline DuckDuckGo/TabViewControllerBrowsingMenuExtension.swift:25]: Small nit - the offsetting `swiftlint enable file_length` can be removed from the end of the file

**Review by amddg44 — APPROVED** [review]
Those changes work perfectly, LGTM! 🙂

**Pull request merged**